### ISSUE TITLE
Theme colours do not have sufficient contrast

### ISSUE THREAD

**Issue by brendanarnold** [DFE-Digital/polis-whitelabel#21]
Form the October 2022 Accessibility review ...

> Your theme colour of white (`FFFFFF`) and blue (`#0494FC`) aren’t accessible unless greater than `18pt`. So you can keep this for your logo – as logo’s are exempt; however for buttons and such you need to choose a higher contrast, so like a darker blue with white text (which I would recommend) – or a paler blue with black text.

**Comment by patcon** [comment]
Hmm, ok, not seeing that color in the upstream main website, so sharing what I'm seeing for clarity :) (scope of this comment might be larger than this issue, but i'll refer back here if needed)

## Logo Color
So the logo is actually `rgb(0, 144, 255)` , which converts to `#3399FF`:
https://convertingcolors.com/rgb-color-0_144_255.html

<img width="387" alt="Screen Shot 2022-12-09 at 5 51 16 PM" src="https://user-images.githubusercontent.com/305339/206763011-69d62679-534a-4b8d-a2a2-f69faabf8071.png">

## Links & Button background

<img width="851" alt="Screen Shot 2022-12-09 at 5 51 26 PM" src="https://user-images.githubusercontent.com/305339/206763046-ae0aad7c-9e9a-4906-b113-8834ea9f80e4.png">

The button background (like "Submit") is `#0090ff`.
The text links (like "more") are `#0a77bf`.

## Admin UI

The behind the scenes admin color (for buttons and header) is `#03a9f4`.

<img width="756" alt="Screen Shot 2022-12-09 at 5 54 40 PM" src="https://user-images.githubusercontent.com/305339/206763467-8d2276e2-e7b3-4a71-8943-b3ec06eb9f19.png">

## Homepage https://pol.is/home

Logo and links on homepage are `#03a9f4`.

<img width="650" alt="Screen Shot 2022-12-09 at 5 56 09 PM" src="https://user-images.githubusercontent.com/305339/206763742-9924ffd0-7795-4cd7-a669-ebfdf115a7de.png">

----

Example convo where I checked: https://pol.is/2demo

**Comment by patcon** [comment]
Can you say which places you are referring to with `#0494FC`? I'm guessing that maybe you already tried to adjust your forks colors for contrast, and that's why it doesn't match the live website? (Or perhaps compdem has recently updated the colors in the codebase, but not yet deployed?)
